DetDescr/src/DetDescr/EcalGeometry.cxx

```diff
@@ -562,10 +520,9 @@ void EcalGeometry::buildNeighborMaps() {
         NNNMap_[center_id].push_back(probe_id);
       }
     }
-    if (verbose_ > 1)
-      std::cout << "  Found " << NNMap_[center_id].size() << " NN and "
-                << NNNMap_[center_id].size() << " NNN for cell " << center_id
-                << std::endl;
+    // ldmx_log(debug) << "  Found " << NNMap_[center_id].size() << " NN and "
+    // << NNNMap_[center_id].size() << " NNN for cell " << center_id
+    ;
```

Comment:
Not sure if you want to uncomment this debug message

Reply:
What was the error you were seeing? We have an `operator<<` defined for it so I thought it would work

https://github.com/LDMX-Software/ldmx-sw/blob/d97b43f5f21da9c6934d2087d00390a782aebb0b/DetDescr/include/DetDescr/EcalID.h#L130